nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -311,3 +311,5 @@ To be able to access the web UI this port needs to be opened in the firewall.
 - Nginx will use the value of `sslTrustedCertificate` if provided for a virtual host, even if `enableACME` is set. This is useful for providers not using the same certificate to sign OCSP responses and server certificates.
 
 - `lib.formats.yaml`'s `generate` will not generate JSON anymore, but instead use more of the YAML-specific syntax.
+
+- Neo4j was updated from version 3 to version 4. See this [migration guide](https://neo4j.com/docs/upgrade-migration-guide/current/) on how to migrate your Neo4j instance.
```

Comment:
You should run [`./nixos/doc/manual/md-to-db.sh`](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/md-to-db.sh) to convert this to docbook, see https://nixos.org/manual/nixos/unstable/#chap-contributing.